server/src/database/entity/chunk.rs

```diff
@@ -44,6 +44,8 @@ pub enum ChunkState {
 pub struct Model {
     /// Unique numeric ID of the chunk.
     #[sea_orm(primary_key)]
+    // https://github.com/SeaQL/sea-orm/issues/1832
+    #[sea_orm(column_type = "Integer")]
     pub id: i64,
```

Comment:
We cannot do this because this would break type mapping on Postgres, resulting in a different schema that's incompatible and incorrect. I suggest holding off and wait for sea-query.

Reply:
https://github.com/SeaQL/sea-query/issues/689 was fixed. I'll do the bump and push this forward.